apps/innovations/_services/innovations.service.ts

```diff
@@ -174,6 +174,121 @@ export class InnovationsService extends BaseService {
     super();
   }
 
+  async archiveInnovation(
+    domainContext: DomainContextType,
+    innovationId: string,
+    data: { message: string },
+    entityManager?: EntityManager
+  ): Promise<void> {
+    const em = entityManager ?? this.sqlConnection.manager;
+
+    const supports = await em
+      .createQueryBuilder(InnovationSupportEntity, 'support')
+      .select([
+        'support.id',
+        'support.status',
+        'support.updatedBy',
+        'userRole.id',
+        'userRole.role',
+        'user.id',
+        'unit.id'
+      ])
+      .innerJoin('support.organisationUnit', 'unit')
+      .leftJoin('support.userRoles', 'userRole')
+      .leftJoin('userRole.user', 'user')
+      .where('support.innovation_id = :innovationId', { innovationId })
+      .getMany();
```

Comment:
Missing condition where support != closed

Reply:
Great catch! Changed in the latest commit.